rclcpp/include/rclcpp/serialization.hpp

```diff
@@ -97,6 +97,34 @@ class Serialization : public SerializationBase
   }
 };
 
+/// Specialized serialization for rcl_serialized_message_t (because type support is not defined)
```

Comment:
I still don't quite understand why we need this specialization. Does the `static_assert` really introduce so much trouble?

Reply:
alternative: you can add enable_if around the functions where it's used

Reply:
I'm also not sure why this is needed.